ginga/tests/test_cmap.py

```diff
@@ -0,0 +1,103 @@
+#
+# Unit Tests for the cmap.py functions
+#
+# Rajul Srivastava  (rajul09@gmail.com)
+#
+import unittest
+import logging
+import numpy as np
+
+import ginga.cmap
+from ginga.cmap import ColorMap
+
+
+class TestError(Exception):
+    pass
+
+
+class TestCmap(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_ColorMap_init(self):
+        test_clst = ((0.0, 0.0, 0.0), (1.0, 1.0, 1.0))
+        test_color_map = ColorMap('test-name', test_clst)
+
+        expected = 'test-name'
+        actual = test_color_map.name
+        assert expected == actual
+
+        expected = 2
+        actual = len(test_color_map.clst)
+        assert expected == actual
```

Comment:
Hi Rajul,

Thanks for making these tests!  ginga color maps have to be of length 256.  It is possible to stretch ones that are smaller to meet this restriction, as is done with the matplotlib colormaps.  But I think a test should create one of the correct size.  It's pretty easy to use `np.linspace()` (for example) to create arrays of the correct length.